config/plugins/interactive_environments/phinch/templates/phinch.mako

```diff
@@ -0,0 +1,49 @@
+<%namespace name="ie" file="ie.mako" />
+<%
+
+import os
+
+# Sets ID and sets up a lot of other variables
+ie_request.load_deploy_config()
+
+# Define a volume that will be mounted into the container.
+# This is a useful way to provide access to large files in the container,
+# if the user knows ahead of time that they will need it.
+user_file = ie_request.volume(hda.file_name, '/home/Phinch/data/'+os.path.basename(hda.file_name)+'.biom', how='ro')
```

Comment:
os.path.join()

How large can a Biom file be (in avarage)?
You are using the mount option here, which is cool and I personally like it very much. But it means that we need to have a shared FS between Dockerhost and Galaxy. We could also copy file via bioblend, like Jupyter, if it's small enough. This means more work but no dependence on a shared-FS.